### PR TITLE
CI

### PR DESCRIPTION
**Pull request by jparisu**
_Branch after #18 _

### CODE REVIEW

**Comment by jparisu** [comment]
CI clang-tidy fails because there is no build for amlip packages yet

File: .github/workflows/ci.repos [hunk 1]
```diff
@@ -0,0 +1,17 @@
+repositories:
```
Comment (juanlofer-eprosima) [inline .github/workflows/ci.repos:1]: amlip missing?

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.repos (hunk 1 above)
Comment (jparisu) [reply .github/workflows/ci.repos:1]: CI.repos does not need to download amlip as it is already taken from github

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**